VideoApi/VideoApi.UnitTests/Services/VirtualRoom/GetOrCreateAnInterpreterVirtualRoomTests.cs

```diff
@@ -108,20 +108,23 @@ public async Task should_create_vmr_with_kinly_if_room_is_not_available()
             
             _mocker.Mock<ICommandHandler>().Setup(x =>
                 x.Handle(It.IsAny<UpdateParticipantRoomConnectionDetailsCommand>())).Callback(() =>
-                expectedRoom.UpdateConnectionDetails(newVmrRoom.Room_label, "ingesturl", newVmrRoom.Uris.Pexip_node,
+                expectedRoom.UpdateConnectionDetails(newVmrRoom.Room_label, "ingesturl_interpreter_2", newVmrRoom.Uris.Pexip_node,
                     newVmrRoom.Uris.Participant));
 
             _mocker.Mock<IKinlyApiClient>().Setup(x => x.CreateParticipantRoomAsync(_conference.Id.ToString(),
                     It.Is<CreateParticipantRoomParams>(vmrRequest => vmrRequest.Participant_type == "Civilian")))
                 .ReturnsAsync(newVmrRoom);
-            
+            var prefix = _conference.IngestUrl;
             var room = await _service.GetOrCreateAnInterpreterVirtualRoom(_conference, participant);
+            var interpreterSuffix = "_interpreter_";
+            var expectedIngestUrl = $"{prefix}{interpreterSuffix}{expectedRoomId}";
 
             room.Should().NotBeNull();
             room.Label.Should().Be(newVmrRoom.Room_label);
             room.PexipNode.Should().Be(newVmrRoom.Uris.Pexip_node);
             room.ParticipantUri.Should().Be(newVmrRoom.Uris.Participant);
-            
+            room.IngestUrl.Equals(expectedIngestUrl).Should().Be(true);
+
```

Comment:
These checks should also be added to GetOrCreateAWitnessVirtualRoomTests since that uses the same methods.